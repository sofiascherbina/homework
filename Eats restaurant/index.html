<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Eats restaurant</title>
    <link href="https://cdn.jsdelivr.net/npm/modern-normalize@2.0.0/modern-normalize.min.css" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="style.css">
</head>
<body> 
    <div class="container">
    <header>
            <div class="header-items">  
                <img class="logo" src="./img/логотип скачать.svg" width="150px" height="65px">
                <div class="navigation">
                    <ul class="navigation-ul">
                        <li class="nav-items">Home</li>
                        <li class="nav-items">Prodact</li>
                        <li class="nav-items">Faq</li>
                        <li class="nav-items">Contact</li>
                    </ul>
                </div>
            </div>
    </header>
    <main>
        <div class="top">
            <h4>Restaurant app</h4>
            <h2>Why stay hungry when <br>you can order form Eats </h2>
            <p>Order in exchange for hunger</p>
                <ul class="buttons-top">
                    <li class="buttons-top-it"><button class="butt-top-el" type="button">Delivery</button></li>
                    <li class="buttons-top-it"><button class="butt-top-el" type="button">Menu</button></li>
                </ul>
        </div>
    </main>
</div>
</body>
</html>