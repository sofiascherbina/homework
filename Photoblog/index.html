<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Photoblog</title>
    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display+SC:ital,wght@0,400;0,700;0,900;1,400;1,700;1,900&family=Playfair+Display:ital,wght@0,400..900;1,400..900&family=Ubuntu:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&display=swap" rel="stylesheet">
    <link
      rel="stylesheet"
      href="https://cdnjs.cloudflare.com/ajax/libs/modern-normalize/1.0.0/modern-normalize.min.css"
      integrity="sha512-ISS7cAi1PEhQ8jnbJpJZMd29NlhNj4AWYyLOSp2CE/CsHxTCu+r+t0D2yoJudVrd0/8fTVPUVDzY5Tvli75u/g=="
      crossorigin="anonymous"
    />
    <link rel="stylesheet" href="./css/style.min.css"/>
</head>
<body>
    <header class="page-header">
        <div class="container">
        <button
        class="menu-toggle js-open-menu"
        aria-expanded="false"
        aria-controls="mobile-menu"
      >
        <svg
          width="24"
          height="24"
          fill="currentColor"
          viewBox="0 0 20 20"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            fill-rule="evenodd"
            d="M3 5a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM3 10a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM3 15a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1z"
            clip-rule="evenodd"
          ></path>
        </svg>
      </button>
            <ul class="menu">
                <li><a href="#" class="link">LIFESTYLE</a></li>
                <li><a href="#" class="link">PHOTODIARY</a></li>
                <li><a href="#" class="link">MUSIC</a></li>
                <li><a href="#" class="link">TRAVEL</a></li>
            </ul>
        </div>
    </header>
    <main>
        <div class="container">
            <ul class="card-ul">
                    <li class="card-ul_el">
                        <img src="./img/1-1.png" alt="More than just a music festival" width="420px" height="280px">
                        <p class="article">LIFESTYLE</p>
                        <h2>More than just a music festival</h2>
                        <p class="lorem">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea.</p>
                    </li>
                    <li class="card-ul_el">
                        <img src="./img/2-1.png" alt="Life tastes better with coffee"  width="420px" height="280px">
                        <p class="article">LIFESTYLE</p>
                        <h2>Life tastes better with coffee</h2>
                        <p class="lorem">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea.</p>
                    </li>
                    <li class="card-ul_el">
                        <img src="./img/3-1.png" alt="American dream"  width="420px" height="280px">
                        <p class="article">PHOTODIARY</p>
                        <h2>American dream</h2>
                        <p class="lorem">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea.</p>
                    </li>
                    <li class="card-ul_el">
                        <img src="./img/4-1.png" alt="A day exploring the Alps"  width="420px" height="280px">
                        <p class="article">PHOTODIARY</p>
                        <h2>A day exploring the Alps</h2>
                        <p class="lorem">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea.</p>
                    </li>
                    <li class="card-ul_el">
                        <img src="./img/1-2.png" alt="Top 10 song for running"  width="420px" height="280px">
                        <p class="article">LIFESTYLE</p>
                        <h2>Top 10 song for running</h2>
                        <p class="lorem">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea.</p>
                    </li>
                    <li class="card-ul_el">
                        <img src="./img/2-2.png" alt="Cold winter days"  width="420px" height="280px">
                        <p class="article">LIFESTYLE</p>
                        <h2>Cold winter days</h2>
                        <p class="lorem">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea.</p>
                    </li>
            </ul>
        </div>
    </main>
    <footer>
        <button class="main-button" type="button">Load more</button>
    </footer>
    <div class="menu-container js-menu-container" id="mobile-menu">
        <button class="menu-toggle js-close-menu">
          <svg
            fill="currentColor"
            viewBox="0 0 20 20"
            xmlns="http://www.w3.org/2000/svg"
            width="24"
            height="24"
          >
            <path
              fill-rule="evenodd"
              d="M4.293 4.293a1 1 0 011.414 0L10 8.586l4.293-4.293a1 1 0 111.414 1.414L11.414 10l4.293 4.293a1 1 0 01-1.414 1.414L10 11.414l-4.293 4.293a1 1 0 01-1.414-1.414L8.586 10 4.293 5.707a1 1 0 010-1.414z"
              clip-rule="evenodd"
            ></path>
          </svg>
        </button>
        <ul class="mobile-menu">
            <li><a href="#" class="link">LIFESTYLE</a></li>
            <li><a href="#" class="link">PHOTODIARY</a></li>
            <li><a href="#" class="link">MUSIC</a></li>
            <li><a href="#" class="link">TRAVEL</a></li>
        </ul>
    </div>
        <script
        defer
        src="https://cdnjs.cloudflare.com/ajax/libs/body-scroll-lock/3.1.5/bodyScrollLock.min.js"
        integrity="sha512-HowizSDbQl7UPEAsPnvJHlQsnBmU2YMrv7KkTBulTLEGz9chfBoWYyZJL+MUO6p/yBuuMO/8jI7O29YRZ2uBlA=="
        crossorigin="anonymous"
      ></script>
      <script defer src="./js/mobile-menu.js"></script>
</body>
</html>