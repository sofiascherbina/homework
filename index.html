<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
    <link rel="stylesheet" href="./style.css">
    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display+SC:ital,wght@0,400;0,700;0,900;1,400;1,700;1,900&family=Playfair+Display:ital,wght@0,400..900;1,400..900&family=Ubuntu:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&display=swap" rel="stylesheet">
    <link href="https://cdn.jsdelivr.net/npm/modern-normalize@2.0.0/modern-normalize.min.css" rel="stylesheet">
</head>
<body>
    <header>
        <div class="container">
            <ul class="topic">
                <li class="topic-li"><a class="topic-el" href="#">LIFESTYLE</a></li>
                <li class="topic-li"><a class="topic-el" href="#">PHOTODIARY</a></li>
                <li class="topic-li"><a class="topic-el" href="#">MUSIC</a></li>
                <li class="topic-li"><a class="topic-el" href="#">TRAVEL</a></li>
            </ul>
        </div>
    </header>
    <main>
        <div class="container">
        <div class="content">
            <ul>
                <div class="container-card">
                    <li class="content-el">
                    <img class="photo" src="./img/1 1.png" alt="More than just a music festival">
                    <p class="content-topic">LIFESTYLE</p>
                    <h2 class="article">More than just a music festival</h2>
                    <p class="lorem">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea.</p>
                    </li>
                </div>
                <div class="container-card">
                    <li class="content-el">
                        <img class="photo" src="./img/2 1.png" alt="Life tastes better with coffee">
                        <p class="content-topic">LIFESTYLE</p>
                        <h2 class="article">Life tastes better with coffee</h2>
                        <p class="lorem">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea.</p>
                    </li>
                </div>
                <div class="carcontainer-card">
                    <li class="content-el">
                        <img class="photo" src="./img/3 1.png" alt="American dream">
                        <p class="content-topic">PHOTODIARY</p>
                        <h2 class="article">American dream</h2>
                        <p class="lorem">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea.</p>
                    </li>
                </div>
                <div class="container-card">
                    <li class="content-el">
                        <img class="photo" src="./img/4 1.png" alt="A day exploring the Alps">
                        <p class="content-topic">PHOTODIARY</p>
                        <h2 class="article">A day exploring the Alps</h2>
                        <p class="lorem">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea.</p>
                    </li>
                </div>
                <div class="container-card">
                    <li class="content-el">
                        <img class="photo" src="./img/1 2.png" alt="Top 10 song for running">
                        <p class="content-topic">LIFESTYLE</p>
                        <h2 class="article">Top 10 song for running</h2>
                        <p class="lorem">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea.</p>
                    </li>
                </div>
                <div class="container-card">
                    <li class="content-el">
                        <img class="photo" src="./img/2 2.png" alt="Cold winter days">
                        <p class="content-topic">LIFESTYLE</p>
                        <h2 class="article">Cold winter days</h2>
                        <p class="lorem">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea.</p>
                    </li>
                </div>
            </ul>
      </div>
    </div>
    </main>
    <footer>
        <button class="main-button" type="button">Load more</button>
    </footer>
</body>
</html>