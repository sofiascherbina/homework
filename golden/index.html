<!DOCTYPE html>
<html lang="uk">
    <head>
        <title>Golden-services</title>
        <meta charset="UTF-8">
        <meta name="keywords" content="">
        <link href="https://cdn.jsdelivr.net/npm/modern-normalize@2.0.0/modern-normalize.min.css" rel="stylesheet">
        <link href="https://fonts.googleapis.com/css2?family=Bilbo+Swash+Caps&family=Montserrat:ital,wght@0,100..900;1,100..900&family=Roboto+Slab:wght@100..900&display=swap" rel="stylesheet">
        <link rel="stylesheet" href="./style.css">
    </head>
    <body>
        <div class="container">
            <header>
                    <a class="logo" href="">Golden</a> 
                    <ul class="navigation">
                        <li class="nav-li"><a href="">Home</a></li>
                        <li class="nav-li"><a href="">Services</a></li>
                        <li class="nav-li"><a href="">Portfolio</a></li>
                        <li class="nav-li"><a href="">About</a></li>
                        <li class="nav-li"><a href="">Contacts</a></li>
                    </ul>
            </header>
            <main>
                <h1 class="title-content">SERVICES</h1>
                <ul class="services">
                    <li class="serv-li">
                        <h3 class="articles">E-Commerce</h3>
                        <p class="description">Proin iaculis purus consequat sem cure<br> digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
                    </li>
                    <li class="serv-li">
                        <h3 class="articles">Responsive Web</h3>
                        <p class="description">Proin iaculis purus consequat sem cure <br>digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
                    </li>
                    <li class="serv-li">
                        <h3 class="articles">Web Security</h3>
                        <p class="description">Proin iaculis purus consequat sem cure<br> digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
                    </li>
                </ul>
            </main>
            <footer>
                <h1 class="title-content">CONTACTS</h1>
                <form>
                    <div class="forms">
                        <label>
                            <input class="forms-el" type="text" name="username" placeholder="YOUR NAME*">
                        </label>
                        <label>
                            <input class="forms-el" type="email" placeholder="YOUR E-MAIL*">
                        </label>
                        <label>
                            <input class="forms-el" type="text" placeholder="SUBJECTS">
                        </label>
                    </div>
                    <label>
                        <textarea class="comment" placeholder="YOUR MESSAGE*" rows="10"></textarea>
                    </label>
                </form>
                <button type="submit">SEND MESSAGGE</button>
            </footer>
        </div>
    </body>
    </html>